api/libproject/cloudsqlproxycontext.rb

```diff
@@ -1,21 +1,17 @@
 require_relative "../../aou-utils/utils/common"
 require_relative "../../aou-utils/workbench"
+require_relative "serviceaccounts"
 
-class CloudSqlProxyContext
-
-  def initialize(gcc)
-    Workbench::assert_in_docker
-    gcc.ensure_service_account
-    @project = gcc.project
-  end
+class CloudSqlProxyContext < ServiceAccountContext
```

Comment:
I don't agree that a CloudSqlProxyContext _is a_ ServiceAccountContext. I would believe that a CloudSqlProxyContext _needs a_ ServiceAccountContext.

Reply:
OK will aim to address in a follow-up.